### PR TITLE
Resolve UI Elements missing or not Correct

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#90300](http://central.tri.be/issues/90300), [C#90274](http://central.tri.be/issues/90274), [C#90272](http://central.tri.be/issues/90272), [C#90276](http://central.tri.be/issues/90276)
---

### CODE REVIEW

File: src/resources/js/tickets-tables.js [hunk 1]
```diff
@@ -14,10 +14,24 @@
 			return;
 		}
 
-		$element.sortable({
+		var fixHelper = function( e, ui ) {
+			ui.children().each( function() {
+				$( this ).width( $( this ).width() );
```
Comment (borkweb) [inline src/resources/js/tickets-tables.js]: Set the width to the width?

File: src/resources/postcss/tickets-tables.pcss [hunk 2]
```diff
@@ -7,13 +7,15 @@
 	border: 1px solid var(--border-color);
 	border-radius: var(--border-radius);
 	margin: 20px 0;
+    border-spacing: 0;
```
Comment (borkweb) [inline src/resources/postcss/tickets-tables.pcss]: Tabbing seems off here. Additionally, this isn't alphabetized.

File: src/resources/postcss/tickets-tables.pcss [hunk 3]
```diff
@@ -46,61 +48,27 @@
...
-	.ui-sortable-handle button {
-		cursor: pointer;
-
-		* {
+	td.column-primary {
+		.tribe-handle {
+			font-size: 1.8em;
+			color: #ccc;
+			position: relative;
+			margin-right: 10px;
```
Comment (borkweb) [inline src/resources/postcss/tickets-tables.pcss]: Alphabetize.

File: src/resources/js/tickets-tables.js (hunk 1 above)
Comment (bordoni) [reply src/resources/js/tickets-tables.js]: To avoid the table collapsing when dragging it around.

**Comment by bordoni** [comment]
Ready for another look.

**Pull request merged**